things/service.go

```diff
@@ -68,6 +70,16 @@ func (svc service) CreateThings(ctx context.Context, token string, cls ...mgclie
 	if _, err := svc.authorize(ctx, "", auth.UserType, auth.UsersKind, user.GetId(), auth.CreatePermission, auth.DomainType, user.GetDomainId()); err != nil {
 		return []mgclients.Client{}, err
 	}
+	ths, err := svc.clients.RetrieveAll(ctx, mgclients.Page{Role: mgclients.AllRole})
+	if err != nil {
+		return []mgclients.Client{}, errors.Wrap(svcerr.ErrViewEntity, err)
+	}
```

Comment:
This should be moved to inside the constraint provider `CheckLimits`

We should have generic repo function in all the service to retrieve `Count` of entitt